compiler/damlc/pkg-db/util.bzl

```diff
@@ -88,7 +79,7 @@ def _daml_package_rule_impl(ctx):
         --write-iface \
         --target {daml_lf_version} \
         -o {dalf_file} \
-        {main}
+        {main} | tail -1 | awk '{{print $3}}' > {pkg_id_file}
```

Comment:
Ah, I guess this is why you print it? Seems somewhat hacky (I think long-term we should change this to call `damlc init` to initialize the bootstrap package db) but given that `compile` is already a hidden command, I’m fine with it for now.

Reply:
Yes, same. It's hacky, happy if we can avoid it in the future.